spring-cloud-zuul-ratelimit-core/src/test/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/repository/ConsulRateLimiterTest.java

```diff
@@ -39,4 +50,23 @@ public void setUp() {
         ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
maybe this `objectMapper` is no longer needed?

Reply:
It is needed for the rest of the tests (from the base class)

Reply:
Got it! I'll merge it then.